backends/gstreamer/lib.rs

```diff
@@ -63,28 +63,50 @@ use servo_media_streams::registry::MediaStreamId;
 use servo_media_streams::MediaOutput;
 use servo_media_traits::{ClientContextId, Muteable};
 use servo_media_webrtc::{WebRtcBackend, WebRtcController, WebRtcSignaller};
-use std::collections::HashMap;
+use std::collections::{HashMap, HashSet};
+use std::hash::{Hash, Hasher};
 use std::path::PathBuf;
 use std::sync::atomic::{AtomicBool, AtomicUsize, Ordering};
 use std::sync::{Arc, Mutex, Weak};
-use std::vec::Vec;
 
 lazy_static! {
     static ref BACKEND_BASE_TIME: gst::ClockTime = { gst::SystemClock::obtain().get_time() };
 }
 
+struct IdMuteable {
+    id: usize,
```

Comment:
`Player`s and `AudioContext`s already have their own ids. I think there is no need to duplicated them here.

Reply:
yes, but we have only `Weak<Mutex<dyn Muteable>>`, is locking mutex just to get an id fine?

Reply:
Hmm, I think that should be fine.

Reply:
ah, this id is also used for Eq + Hash!